src/index.json

```diff
@@ -58504,6 +58504,66 @@
                     "version": "0.2.4"
                 },
                 "sha256Digest": "daca7562e92ddeb0195d2e85bdad22f0873010359ada146b1bfd8e415244cac9"
+            },
+            {
+                "downloadUrl": "https://github.com/Azure/partnercenter-cli-extension/releases/download/v0.2.6-alpha/partnercenter-0.2.6-py3-none-any.whl",
```

Comment:
cli extension released new versioning schema, which asks for minimum stable version to be 1.0.0 and preview version to be 1.0.0b1.

For more info, please refer to https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#notes-1

Reply:
Thank you.  Noted.  Will change in next release.